lib/compat/wordpress-6.5/block-bindings/post-meta.php

```diff
@@ -17,7 +17,7 @@ function gutenberg_block_bindings_post_meta_callback( $source_attrs, $block_inst
 		return null;
 	}
 
-	$post_id = isset( $block_instance->context['postId'] );
+	$post_id = $block_instance->context['postId'];
```

Comment:
Should it still return `null` when the context is not set? We guard many core blocks with a similar check inside `render_callback`:

```php
if ( ! isset( $block->context['postId'] ) ) {
	return null;
}
```

Example:

https://github.com/WordPress/gutenberg/blob/7d750526d7d4ea2020350239e40cc93e575b8caa/packages/block-library/src/post-title/index.php#L19-L22

Reply:
I made this change in [this commit](https://github.com/WordPress/gutenberg/pull/58554/commits/0534638a4add3261c821ac38fde95faf4f77d000). I just realized that I used `! isset` instead of `empty` because I copied and pasted. Is `empty` better here?

Reply:
Both should work fine. `empty` is stronger as it runs `isset` plus ensures that the value is not falsy which is not likely to happen anyway.